Sources/PackageCollections/API.swift

```diff
@@ -0,0 +1,179 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import PackageModel
+import SourceControl
+
+public protocol PackageCollectionsProtocol {
+    // MARK: - Package set profile APIs
```

Comment:
```suggestion
    // MARK: - Package collection profile APIs
```

We might also consider renaming profile to group now that we call these PackageCollection instead of PackageGroup but let's leave that for later.